### ISSUE TITLE
 Step out sometimes does not stop the target after the function returns

### ISSUE THREAD

**Issue by xusheng6** [microsoft/WinDbg-Feedback#86]
For a x86 binary, suppose the call to a function is followed by a "jmp" instruction, if I step into that function and wish to step out from it, the target does not stop after it returns from the function. The target continues to execute freely as if I send a "go" command instead.

The bug can be reproduced by debugging this binary: https://github.com/Vector35/debugger/blob/dev/test/binaries/Windows-x86/helloworld.exe. Its entry point looks like this:

```
_start:
004012d5  call    ___security_init_cookie
004012da  jmp     sub_401153
```

Steps to reproduce:

1. Debug the binary with WinDbg or WinDbg preview
2. Put a breakpoint at 0x4012d5  (remember to deal with ASLR and calculate the actual address)
3. Resume the target and let the breakpoint hit
4. Press "Step Into" button
5. The target should stop at the first instruction of __security_init_cookie, 0x40157d
6, Press "Step Out" button
7. Observe the target runs freely and then exits, instead of stopping at 0x4012da  

Note this only reproduces on a x86 binary. I tested a x64 binary with a similar "jmp-after-call" setting but WinDbg works correctly.

The bug is tracked on our end by https://github.com/Vector35/debugger/issues/421.One of our customers reported the bug to me, and after triaging, I figured out this is a bug in the underlying dbgeng. So I am letting the WinDbg team know about it.

**Comment by KenSykes** [comment]
We are cleaning up the issues list. This bug is older than six months. If you still see this issue in the current debugger and would like to see a fix please submit a new report.\ Thanks!

**Comment by xusheng6** [comment]
> We are cleaning up the issues list. This bug is older than six months. If you still see this issue in the current debugger and would like to see a fix please submit a new report.\ Thanks!

Yeah I will look into this and re-file if appropriate. Is this repo still the best place to file a bug report or feature request? Or there is now a new/different way to do so?

**Comment by KenSykes** [comment]
Yes, this is still the right place to re-file, and we are cleaning it up and paying more attention these days. Thanks for re-filing the report.
